app/controllers/concerns/form_attachment_create.rb

```diff
@@ -55,8 +55,10 @@ def save_attachment_to_cloud!
     form_attachment.set_file_data!(filtered_params[:file_data], filtered_params[:password])
   rescue => e
     log_message_to_sentry(
-      'form attachment error 2',
+      'form attachment error 2 - save to cloud',
       :info,
+      has_pass: filtered_params[:password].present?,
```

Comment:
Any reason not to just make this `has_password`?

Reply:
I'm a little worried about the content being filtered based on the key - already ran in to that with `class_name:`'s content being redacted for me.

Not actually sure what the redaction rules look like, actually?

Reply:
Gotcha, I guess I hadn't seen that yet. Looks fine to me then 👍